Index: themes/beautiful-hexo/_config.yml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- themes/beautiful-hexo/_config.yml	(revision 2e13bd07fcc96d6ab50566349a312ccb01fa5e6e)
+++ themes/beautiful-hexo/_config.yml	(revision )
@@ -1,5 +1,4 @@
 navbar-links:
-    Archive: /archives
     Github: https://github.com/updaterllc
     About: /about/
 
@@ -48,7 +47,7 @@
     link: '/atom.xml'
     title: RSS
     icon: fa-rss
-    enable: true
+    enable: false
   twitter:
     link: 'https://twitter.com/sato1043'
     title: Twitter
Index: themes/beautiful-hexo/layout/index.jade
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- themes/beautiful-hexo/layout/index.jade	(revision 2e13bd07fcc96d6ab50566349a312ccb01fa5e6e)
+++ themes/beautiful-hexo/layout/index.jade	(revision )
@@ -8,33 +8,3 @@
 
     include partial/what-we-do
 
-    div(class="container", role="main")
-        .row
-            .col-lg-8.col-lg-offset-2.col-md-10.col-md-offset-1
-                .posts-list
-                    - page.posts.each(function (post) {
-                    article.post-preview
-                        a(href=url_for(post.path))
-                            h2.post-title!= post.title
-                            if post.subtitle
-                                h3.post-subtitle!= post.subtitle
-                        include partial/post-meta
-                        +post_meta(post.date, post.tags)
-                        .post-entry!= post.excerpt
-                        a(href=url_for(post.path), class="post-read-more")!= __('more')
-                    - })
-                ul.pager.main-pager
-                    if page.prev
-                        li.previous
-                            a(href=url_for(page.prev === 1 ? "/" : ("/" + config.pagination_dir + "/" + page.prev)))= __('prev')
-                    if page.next
-                        li.next
-                            a(href=url_for("/" + config.pagination_dir + "/" + page.next))= __('next')
-
-
-
-
-
-
-
-
